<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainScreen</class>
 <widget class="QWidget" name="CMainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMainScreen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tabDevice">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>当前设备</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tabSoftware">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>可用升级包</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnUpdate">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>开始升级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="KeyReset">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>重启设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRun">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>运行程序</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>删除升级包</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExit">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textFile">
           <property name="text">
            <string>/mnt/nfs/FreeRTOS.bin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
